client/components/draft_input/draft_input_caption.coffee

```diff
@@ -0,0 +1,204 @@
+React = require 'react'
+window.global = window
+window.process = {env: {NODE_ENV: 'development'}}
```

Comment:
Oh Draft.js—a only-useful-in-the-browser library that implicitly depends on Node.js globals.